src/main/features/dashboard/views/claimant.njk

```diff
@@ -16,7 +16,20 @@
     <div class="column-two-thirds">
       <div>
         <h2 class="heading-medium">{{ t('Claim status') }}</h2>
-        {{ responseStatusForClaimant(claim, DashboardUrlHelper.getStatusUrl(claim)) }}
+        {{ statusForClaimant(claim, DashboardUrlHelper.getStatusUrl(claim)) }}
+
+        <p>
```

Comment:
what is this?

Reply:
Its the same macro just renamed as it is more than just a status about a response

Reply:
why is there a macro there twice then?

Reply:
Are you referring to "claimantRespondToOffer"?
I don't see it above in your highlight.

Reply:
yes

Reply:
I understand now.
statusForClaimant is the status that a claimant will see in the dashboard and the claimant.njk.
claimantRespondToOffer is the clickable "View their offer." link in the claimant page (That was removed from the status)